src/test/java/LoginTests.java

```diff
@@ -0,0 +1,70 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+
+public class LoginTests extends BaseTest {
+    @Test
+    public void LoginWrongCredentialTest() throws InterruptedException {
+
+        // openLoginPage();
+        //enterEmail("europlatinum@mail.ru");
+        //enterPassword("Newjob2!");
+        driver.get("https://kwidos.com/auth/login");
+        driver.findElement(By.cssSelector("#email")).sendKeys("europlatinum@mail.ru");
+        Thread.sleep(4000);
```

Comment:
Please use "new WebDriverWait(...).until(..)" instead of "Thread.sleep()". And please fix it everywhere.